CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+# How to contribute
+
+## Prerequisites
+
+- [Node.js][n] 16+;
+- [Git][g];
+- [Docker Desktop][d];
+- IDE:
+  - JetBrains — any with the [Node.js plugin][np], as example [Webstorm][ws];
+  - or Microsoft [Visual Studio Code][vsc];
+- Installed and enabled IDE plugins:
+  - EditorConfig;
+  - ESLint;
+  - Prettier.
+
+[n]: https://nodejs.org/
+[g]: https://git-scm.com/
+[d]: https://www.docker.com/products/docker-desktop
+[np]: https://plugins.jetbrains.com/plugin/6098-node-js
+[ws]: https://www.jetbrains.com/webstorm/
+[vsc]: https://code.visualstudio.com/
+
+## Setup
+
+1. [Fork][f] the Storybook builder vite repository.
+
+   [f]: https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+2. Clone forked repository:
+
+   ```
+   git clone https://github.com/YOUR_USERNAME/storybook-builder-vite
+   cd storybook-builder-vite
+   ```
+
+3. Install dependencies:
+
+   ```
+   yarn
+   ```
+
+## Setup JetBrains IDE
```

Comment:
I don't think this section belongs here, personally.  This is specific to an editor, not this project, and is liable to become out of date, e.g. if other file types are added or other files need to be excluded.

Reply:
I can remove this section.

Reply:
Fixed.